### DISCUSSION TITLE
[Scam Alert] DeFI Groups دیفای گروپس

### DISCUSSION THREAD

**Discussion by Webamoozcom** [Webamoozcom/warning-list#39] [Announcements]
<div dir="rtl">

**Project Name:** 
DEFI Groups

**نام پروژه:** 
دیفای گروپس

**نماد پروژه:**
JKR, DefIX

**اتفاقی که افتاده است:**
شرکت ناشناسی بدون آدرس و شناسنامه و هویت ادعای شرکت دیفای(مالی غیر متمرکز) دارد

1.دیفای اصلا #رفرال_گیری نداره
2.سود Stake کردن ماهی 10 تا 17 درصد نیست خیلی کمتره
3.برای Stake کردن شما پکیج سرمایه گذاری نمیخری
این پروژه قطعا کلاهبرداریه
یک پروژه ی پانزی رو به اسم دیفای معرفی کردند

تو تبلیغاتش داره 360 درصد سوددهی خالص رو‌ پروموت میکنن

https://t.me/Defi_2022/147
https://t.me/Defi_2022/145

**قوانینی که نقض شده است:**

**نوع هشدار:**
هشدار کلاهبرداری

**نوع کلاهبرداری:** 
طرح پانزی و ریفرال

**وضعیت کلاهبرداری:**
در حال انجام

**نام کلاهبرداران:**

**نام شرکت(ها):**
DeFi network is developed by DeFI Consulting 

**آدرس وب سایت(ها):**
https://defigroups.com/
https://t.me/Defi_2022/147
https://t.me/Defi_2022/145

**برآورد مبلغ کلاهبرداری:**

**منابع:**

**جمع بندی:**

**Comment by sonofwisdom2** [comment]
این افراد به صورت کاملا پانزی مشغول جمع آوری پول از مردم هستن و حتی شماره تماس شون هم مشخصه. 
یکی از لیدرهای به نام این جریان سحر صابری هستش. 
[Forwarded from مشاور مالی]
@Defi_ml
09017700575
مهندس مهرداد از تهران ، ، یکی از لیدرهاشون 
خانم لیلا با شماره تماس: 09136549890
آقای علیرضا شادمانی از لیدرهای تهران و قزوین: 
09100443624
امیدوارم دستگاه قضایی چشمش رو به این جنایت اقتصادی نبنده و تا دیر نشده بره سر وقت این ها و این آدم ها هم جوابگو باشند.

**Comment by mehran-mrn** [comment]
اطلاعات who.is ثبت کننده این سه دامنه یکسان هست
https://icbroker.com/ 
 https://waykingroup.com/
https://defigroups.com/

اطلاعات ثبتی whois:

Address: Kalkofnsvegur 2
City: Reykjavik
State / Province: Capital Region
Postal Code: 101
Country: IS
Phone: +354.4212434

و این اطلاعات ثبت کننده دامنه در گزارش های زیادی در خصوص ارسال بدافزار و باج افزار اعلام شده است
از جمله گزارش CISA که در این گزارش اعلام شده از دو دامنه با همین اطلاعات ثبت کننده به سیستم کاربران باج افزار ارسال شده 
لینک گزارش:
https://us-cert.cisa.gov/sites/default/files/publications/MAR-10337802-1.v1.WHITE.pdf
